k8s/data-pipeline/services/oauth2-proxy.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: oauth2-proxy
```

Comment:
Similar to above, is this label necessary here?

Reply:
Removed.